### ISSUE TITLE
Add bidi suport for beamer class

### ISSUE THREAD

**Issue by seloumi** [bidi-tex/bidi#2]
Package bidi addapt many packages and classes for right to left writing and It does well. But why not with  beamer?  Are there obstacles to do so ( with xelatex)?  It would be great if there a support of beamer with bidi.

**Comment by davidcarlisle** [comment]
Yes although in practice we might need explicit code change suggestions from someone who is familiar with these scripts.

Also Vafa Kha­lighi, who did most of the development of these packages before handing over to this group, has found time to do some further updates, so you could ask if he can add something (I would sync any changes back here if he did)  https://github.com/bidirtex

**Comment by davidcarlisle** [comment]
See https://github.com/bidirtex/bidi/issues/4
